source/common/http/conn_manager_impl.h

```diff
@@ -361,34 +361,25 @@ class ConnectionManagerImpl : Logger::Loggable<Logger::Id::http>,
 
   using ActiveStreamEncoderFilterPtr = std::unique_ptr<ActiveStreamEncoderFilter>;
 
-  // Used to abstract making of RouteConfig update request.
-  // RdsRouteConfigUpdateRequester is used when an RdsRouteConfigProvider is configured,
-  // NullRouteConfigUpdateRequester is used in all other cases (specifically when
```

Comment:
Why was this pattern changed?

Reply:
Because the requester now hold the responsibility of requesting both srds and vhds update, we still need a requester when scope_rds is used.(In the past, a null_requester is initialized when scoped_rds is used).